nuxeo-features/nuxeo-admin-center/nuxeo-admin-center-core/src/main/java/org/nuxeo/ecm/admin/operation/HotReloadStudioSnapshot.java

```diff
@@ -211,6 +228,89 @@ public Blob hotReloadPackage(DownloadablePackage remotePkg) {
 
     }
 
+    public Blob hotReloadPackage(String packageId) {
```

Comment:
missing since annotation, but actually, does this need to be a public method?

Reply:
Not really, let's keep it like this right now as I will rework this as a service/helper to bridge admin center too.